test/test_support.ts

```diff
@@ -279,56 +243,63 @@ function extractInlineSourceMap(source: string): string {
     result = inlineSourceMapRegex.exec(source);
   } while (result !== null);
   const base64EncodedMap = previousResult![1];
-  return Buffer.from(base64EncodedMap, 'base64').toString('utf8');
+  const sourceMapJson = Buffer.from(base64EncodedMap, 'base64').toString('utf8');
+  return sourceMapTextToConsumer(sourceMapJson);
 }
 
-function getFileWithName(filename: string, files: Map<string, string>): string|undefined {
-  for (const filepath of toArray(files.keys())) {
+export function getFileWithName(filename: string, files: Map<string, string>): string {
```

Comment:
`getFileContents...`? `findFileContentsByName`? This could also use docs, it's not obvious from the function name that `withName` means basename in this case.